### ISSUE TITLE
Sigin Signup Not working

### ISSUE THREAD

**Issue by viditpagi123** [adrianhajdin/social_media_app#48]
While Done with the signin and signup part after doing both activities its giving failed to sign in  and Signup

**Comment by Chirag-2003** [comment]
well a solution that works is you have to downgrade your appwrite,
just go to terminal and run the following command -
npm install appwrite@13.0.2
It'll work, had the same issue as you

**Comment by viditpagi123** [comment]
Yup did the same and it's working fine now

**Comment by viditpagi123** [comment]
> well a solution that works is you have to downgrade your appwrite, just go to terminal and run the following command - npm install appwrite@13.0.2 It'll work, had the same issue as you

Bro Posts are not getting saved in appwrite

**Comment by KasakAnand07** [comment]
In AuthLayout.tsx page side-img.svg is not working. Can you please help me to solve this issue?

**Comment by anantiikenna** [comment]
Appwrite has been updated  so the former 
 `account.createEmailSession(user.email, user.password)` 
 
 has change to 
`account.createEmailPasswordSession(user.email, user.password)`.

so the updated code for signInAccount  in the api.ts file will be 

```

export async function signInAccount(user: {email: string; password: string;}) { 
  try {
    const session = await account.createEmailPasswordSession(user.email, user.password);

    return session;
      
  } catch (error) {
      console.log(error);
      return error;
  }
}

```

**Comment by AnuOdinson117** [comment]
> Appwrite has been updated so the former `account.createEmailSession(user.email, user.password)`
> 
> has change to `account.createEmailPasswordSession(user.email, user.password)`.
> 
> so the updated code for signInAccount in the api.ts file will be
> 
> ```
> 
> export async function signInAccount(user: {email: string; password: string;}) { 
>   try {
>     const session = await account.createEmailPasswordSession(user.email, user.password);
> 
>     return session;
>       
>   } catch (error) {
>       console.log(error);
>       return error;
>   }
> }
> ```

**Comment by AnuOdinson117** [comment]
@anantiikenna Yes you are right they did change the syntax but the routing from sign-up to root is non-functional. Many others things are needed to be changed for that.

So that code didn't work for me (thesis for at least 3 hours) and the conclusion is it wont work. So its better to go with the previous method of installing the older Appwrite version.

```npm i appwrite@13.0.2```

Have a nice day!

**Comment by anantiikenna** [comment]
Mine works, you can check this link for reference
https://github.com/anantiikenna/socio_max

**Comment by AnuOdinson117** [comment]
@anantiikenna Saw it .. i noticed you changed the theme color from lavender to orange .. its cool

**Comment by anantiikenna** [comment]
> Appwrite has been updated so the former `account.createEmailSession(user.email, user.password)`
> 
> has change to `account.createEmailPasswordSession(user.email, user.password)`.
> 
> so the updated code for signInAccount in the api.ts file will be
> 
> ```
> 
> export async function signInAccount(user: {email: string; password: string;}) { 
>   try {
>     const session = await account.createEmailPasswordSession(user.email, user.password);
> 
>     return session;
>       
>   } catch (error) {
>       console.log(error);
>       return error;
>   }
> }
> ```

Thanks

**Comment by anantiikenna** [comment]
> @anantiikenna Saw it .. i noticed you changed the theme color from lavender to orange .. its cool

Thanks

**Comment by adityaranjan2005** [comment]
@anantiikenna, editor isn't recognizing createEmailPasswordSession

![image](https://github.com/user-attachments/assets/0a77da58-872e-48ef-bf3a-cda0308c6037)

**Comment by anantiikenna** [comment]
mine uses the updated Appwrite v14.0.1
and kindly share the error message it generated on your console.

**Comment by AnuOdinson117** [comment]
> @anantiikenna, editor isn't recognizing createEmailPasswordSession
> 
> ![image](https://private-user-images.githubusercontent.com/145968304/358215856-0a77da58-872e-48ef-bf3a-cda0308c6037.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM4NDI3MDQsIm5iZiI6MTcyMzg0MjQwNCwicGF0aCI6Ii8xNDU5NjgzMDQvMzU4MjE1ODU2LTBhNzdkYTU4LTg3MmUtNDhlZi1iZjNhLWNkYTAzMDhjNjAzNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgxNlQyMTA2NDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iYmIzMjU1MTA1Y2VlNTg5YTMxNzhjMDMyZWMwYzdhNjUxZGY5NjYyNGFiNzdjNGI3NTRhM2NlYTNhMGRjOGNjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.wwlFrpZksQ_9DhqBgBt4SBIENBnrz1E1ov062V-NyQU)

Switch back to a previous Appwrite Version as it used .createEmailSession and modern one is .createEmailPasswordSession but there is some issue with making it work with Appwrite Storage. 

Go to terminal and do
```npm i appwrite@13.0.2```

**Comment by AdugboAghogho** [comment]
> Mine works, you can check this link for reference https://github.com/anantiikenna/socio_max

where u able to fix the signin/sign-up after deployment..?

**Comment by anantiikenna** [comment]
> > Mine works, you can check this link for reference https://github.com/anantiikenna/socio_max
> 
> where u able to fix the signin/sign-up after deployment..?

Yes it fix recently updated.
Planning to shift to PostgreSQL because Appwrite image restrictions.
